src/ModalContent/ModalContent.stories.js

```diff
@@ -0,0 +1,89 @@
+import AriaModal from 'react-aria-modal';
+import React, { useState } from 'react';
+import { ModalContent } from './ModalContent';
+import { Button } from '../Button/Button';
+
+export default {
+  title: 'Composite/Modal Content',
+  component: ModalContent,
+  subcomponents: {
+    'ModalContent.Title': ModalContent.Title,
+    'ModalContent.Body': ModalContent.Body,
+    'ModalContent.Cta': ModalContent.Cta,
+    'ModalContent.Footer': ModalContent.Footer,
+  },
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/Kmd4utmJFPRMVeCFEEBQhLtx/Decanter-Design-System?node-id=16530%3A326',
+    },
+    docs: {
+      description: {
+        component: 'More information coming soon...',
+      },
+    },
+  },
+};
+
+const ModalContentTemplate = ({ onClose, ...rest }) => (
+  <ModalContent {...rest} onClose={onClose}>
+    <ModalContent.Title>
+      <h2 className="su-h2">Title</h2>
+    </ModalContent.Title>
```

Comment:
Might as well make the ModalContent.Title component what is in the contents here. No sense in adding the extra divs.  

Edit: Remove Modal content title and replace the `h2` with the Heading component: https://deploy-preview-64--decanter-react.netlify.app/?path=/story/simple-heading--h-2